<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StorageWidget</class>
 <widget class="QWidget" name="StorageWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>300</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid black; padding: 5px; background: rgb(30, 0, 0)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="headline">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MoolBoran</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0; color: white;</string>
   </property>
   <property name="text">
    <string>Storage</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="storageClose">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>10</y>
     <width>21</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0; color: white; background: rgb(30, 0, 0)</string>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>361</width>
     <height>221</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid black; background: rgb(20, 0, 0)</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <zorder>listWidget</zorder>
  </widget>
  <widget class="QLabel" name="labelFull">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0; color: white;</string>
   </property>
   <property name="text">
    <string>0% full</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>162</x>
     <y>40</y>
     <width>187</width>
     <height>231</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid black; color: white; background: rgb(30, 0, 0);font-size: 8pt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="acceptedResourcesButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>12</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid black; color: white; background: rgb(30, 0, 0); font-size: 8pt;</string>
   </property>
   <property name="text">
    <string>All Resources Accepted</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
